specs/swagger.yml

```diff
@@ -2892,6 +2892,27 @@ paths:
           $ref: '#/responses/unauthorizedResponse'
         '409':
           $ref: '#/responses/cannotDeleteReferencedResourceResponse'
+
+  ###################################################################
+  # Database
+  ##################################################################
+  '/database/snapshot':
+    post:
+      summary: Create a new database snapshot
+      description: Create a new database snapshot. Requires admin access.
+      security:
+        - ztSession: []
+      tags:
+        - Database
+      operationId: createDatabaseSnapshot
+      responses:
+        '200':
+          $ref: '#/responses/emptyResponse'
+        '400':
+          $ref: '#/responses/badRequestResponse'
```

Comment:
If there are no inputs to this request, I don't believe it can return 400 bad request.

Reply:
Makes sense. It can fail if you invoke more than one per minute. That probably doesn't count as a bad request, though.

Reply:
I suggest we detect that and throw a 409 conflict

Reply:
I'll have to put in a fabric change to make the error detectable